opentofu/misc.tf

```diff
@@ -11,4 +11,7 @@ resource "google_compute_project_metadata" "enable_oslogin" {
     enable-oslogin     = "TRUE"
     enable-oslogin-2fa = "TRUE"
   }
+  lifecycle {
+    ignore_changes = [metadata]
+  }
 }
```

Comment:
This is done to avoid wiping out variables that aren't explicitly set in the given function

Reply:
can you clarify what variables these are? I'm not convinced of this change

Reply:
We had a talk about this face to face, but for long term documentation:

This change is due to having one gcloud project shared by multiple services that we host, this makes it so that we don't screw with the variables of other services whenever we deploy this one
Without this block, we wipe out all prior metadata and implant the new stuff; this preserves the prior and adds the new, though with risk of overwriting some of the old if the same variables are used with different values elsewhere